master/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: Master chart that deploys the OS. Targets minikube 
-name: master
+name: platform
```

Comment:
If we're going to rename this chart we should also rename the files and directory.